sql/schema.sql

```diff
@@ -15,7 +15,9 @@ DROP TABLE IF EXISTS `ghtorrent`.`users` ;
 CREATE TABLE IF NOT EXISTS `ghtorrent`.`users` (
   `id` INT(11) NOT NULL AUTO_INCREMENT COMMENT '',
   `login` VARCHAR(255) NOT NULL COMMENT '',
+  `name` VARCHAR(255) NULL DEFAULT NULL COMMENT '',
```

Comment:
Right.  The challenge however is that someone setting up their own GHT will need to define the schema in their database.  That instance will have email and name fields. The code fails if they are not in the schema. We actually want these values as we're watching our own repos so the values are quite reasonable to harvest and we'll take care to use them properly.

The thinking here is thta it is ok to define columns that are empty all the time in some data sets so having them in the schema is not a particular problem.